manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -291,6 +291,7 @@ metadata:
     name: "{{.Name}}"
     uid: "{{.UID}}"
 spec:
+  ipFamilyPolicy: PreferDualStack
```

Comment:
I think it's good to `Prefer` for Istio services, where we know for a fact it won't interact badly with user apps that may be `SingleOnly` or `Require`.

My only concern with this is that it might make debugging issues a bit harder, since it's a (afaik) silent fallback that could lead to subtly different behavior, depending on the env.

I dunno how valid that is tho, and we may not be able to do much about it.

Reply:
Fallback is entirely up to the client. Its common to use happy-eyeballs fallback algorithm though. I agree it can cause some concerns there. 

From ztunnel itself we won't do fallback so that part shouldn't be a problem at least

Reply:
I donot see much harm other than dual stack feature is experimental